common/views/components/Modal/Modal.tsx

```diff
@@ -212,15 +212,14 @@ const Modal: FunctionComponent<Props> = ({
   useEffect(() => {
     function closeOnEscape(event: KeyboardEvent) {
       if (event.key !== 'Escape') return;
-
       closeModal();
     }
-    if (!removeCloseButton) {
+    if (!removeCloseButton && isActive) {
```

Comment:
would the isActive check be better inside the closeOnEscape function, rather than using it to add the eventListener?

Reply:
I think it would have to be passed as an argument to the `closeOnEscape` function, so not sure there would be a benefit.